src/SimpleGraphs/generators/randgraphs.jl

```diff
@@ -1045,3 +1045,27 @@ function dorogovtsev_mendes(n::Integer; seed::Int=-1)
     end
     return g
 end
+
+"""
+    random_orientation_dag(n)
+
+Generate a random oriented acyclical digraph. The function takes in a simple/
+graph and an rng as an argument. The probability of each random dag being generated depends/
+the architecture of the original directed graph.
+
+DAG's have a finite topological order; this order is randomly generated via "order = randperm()". 
+"""
+function random_orientation_dag(g::SimpleGraph{T}, rng = GLOBAL_RNG) where T <: Integer
+    nv = length(g.fadjlist)
+    tnv = T(nv)
+    order = randperm(rng, tnv)
+    g2 = SimpleDiGraph(tnv) #initialize empty adj lists for each node; both f and b
+    for i in (1 : tnv)
+        for j in 1:outdegree(g, i)
+            if (order[i] < order[g.fadjlist[i][j]])
```

Comment:
don't use `fadjlist` here. You're trying to get the `j`th neighbor of `i`, and you're not using `j` as anything but an index, so just iterate over the neighbors:

(This code replaces what you have starting at 1059):
```
g2 = SimpleDiGraph(nv(g))
@inbounds for i in vertices(g)
  for j in outneighbors(g, i)
    if order[i] < order[j]
...
```

You don't need parentheses for loops or for conditionals.

Reply:
Thanks, we made the requested change.